federation/pkg/federation-controller/service/dns/dns.go

```diff
@@ -203,20 +204,20 @@ func (s *ServiceDNSController) validateConfig() error {
 		return fmt.Errorf("DNSController should not be run without federationName")
 	}
 	if s.zoneName == "" && s.zoneID == "" {
-		return fmt.Errorf("DNSController must be run with either zoneName or zoneID")
+		return errors.New("DNSController must be run with either zoneName or zoneID")
```

Comment:
In fact, fmt.Errorf equals errors.New, and if you want to use errors.New(), you should inport "errors".
So it seems to be unncessary.